hazelcast/src/main/java/com/hazelcast/internal/serialization/impl/PriorityBlockingQueueStreamSerializer.java

```diff
@@ -43,11 +43,8 @@ public void write(ObjectDataOutput out, PriorityBlockingQueue<E> collection) thr
     @Override
     public PriorityBlockingQueue<E> read(ObjectDataInput in) throws IOException {
         Comparator<E> comparator = in.readObject();
-
         int size = in.readInt();
```

Comment:
How can it be that the value 0 was received?

Reply:
If an empty priorityQueue was saved.